### DISCUSSION TITLE
Not able to view events in resources dynamically

### DISCUSSION THREAD

**Discussion by Vinita6484** [vkurko/calendar#259] [Q&A]
![image](https://github.com/vkurko/calendar/assets/85390301/30e7fed3-311f-4913-a821-096f0401a858)
this is the image of a week grid where events are views. and below the calender is my dynamic json which I am passing.
![image](https://github.com/vkurko/calendar/assets/85390301/4481fdf7-15cf-4878-9119-2a4a93570987)
Not able to view events in the resourcegrid

**Comment by vkurko** [comment]
Please provide an online demo of your code in [JS Bin](https://jsbin.com/?html,css,output) or JS Fiddle/Codepen/etc.
